packages/dataviews/src/components/dataviews/index.tsx

```diff
@@ -112,6 +114,44 @@ export default function DataViews< Item >( {
 		( filters || [] ).some( ( filter ) => filter.isPrimary )
 	);
 
+	const defaultUI = (
+		<div className="dataviews-wrapper" ref={ containerRef }>
```

Comment:
Should we keep the wrapper even if `children` are passed? Or alternatively add it in the `DataViewsLayout` component possibly?

The width from `containerRef` is used in `PreviewSizePicker` to dynamically calculate breakpoints and column numbers in `grid` layout. I didn't test, but I wouldn't expect it work properly right now with `free composition`.

Reply:
It's a good question. What is the responsibility of the DataViews in terms of UI in the free composition mode?

In this case, it should be addressed internally, but at a different level, probably in the Layout component.